.sample/README.md

```diff
@@ -9,9 +9,9 @@ tags: [community]
 
 # MODULE_NAME
 
-<-- Describes what this module does -->
+<!-- Describes what this module does -->
```

Comment:
Have you verified that this is valid markdown comment now?
I think we need three dashes instead of 2.